doc/decisions/commit_function.md

```diff
@@ -1,6 +1,11 @@
 # Commit Function
```

Comment:
You can set this decision back to draft and add the options normally (not commented out). A related decision is if "returning the contract" should be its own function.